sass/_forms.scss

```diff
@@ -44,6 +44,11 @@
             font-size: var(--font-size);
         }
 
+        .clear-pres-dialog{
+            margin-top : 0.75em;
+            margin-right: 0.2em;
+        }
```

Comment:
Why don't you reuse the `.clear-input` class?

Reply:
I see it's being used in `src/templates/chat_status_modal` and in `src/templates/roster_filter.html`

Reply:
I realized that clear-input also controls the same icon for filtering contacts. But as the placeholders for the presence dialog and the roster filter are of different sizes, using only clear-input will make the clear cross not well situated in the placeholder for the roster filter.

Reply:
Can you not use CSS specificity rules to overcome this? Sass lets you nest rules within one another, you can probably use that to style `.clear-input` differently for each use-case.